ext_emconf.php

```diff
@@ -8,10 +8,10 @@
     'author_email' => 'info@pixelant.net',
     'author_company' => 'Pixelant',
     'state' => 'stable',
-    'version' => '1.0.3',
+    'version' => '2.0',
     'constraints' => [
         'depends' => [
-            'typo3' => '8.7.0-9.5.99'
+            'typo3' => '10.4.99'
```

Comment:
State the lower version requirement explicitly here too.
```suggestion
            'typo3' => '10.4.0-10.4.99'
```